src/drt/src/dr/FlexDR.cpp

```diff
@@ -1823,6 +1824,43 @@ void FlexDR::searchRepair(const SearchRepairArgs& args)
                   199,
                   "  Number of violations = {}.",
                   getDesign()->getTopBlock()->getNumMarkers());
+    if (getDesign()->getTopBlock()->getNumMarkers() > 0) {
+      // report violations
+      std::map<std::string, std::map<frLayerNum, uint>> violations;
+      std::set<frLayerNum> layers;
+      const std::map<std::string, std::string> relabel
+          = {{"Lef58SpacingEndOfLine", "EOL"},
+             {"Lef58CutSpacingTable", "CutSpcTbl"},
+             {"Lef58EolKeepOut", "eolKeepOut"}};
```

Comment:
There is no need to remap here since you can just change the name in getViolName()  [my script didn't have the choice]

Reply:
The same getViolName function is used in the drcReport. I don't want to change the names there.

Reply:
I don't like having two different sets of names in different reports.   I think we should have a compact name for reporting and in the drc report we should have the lef file/line# if someone wants the full details of the constraint (it wouldn't be hard to keep that in odb).

Reply:
I am not sure if it's easy to get the line number of the rule from the lef in odb, but I think it would require me to edit the "lef" module in odb to keep track of the line numbers for the properties. Even if it's easy, we'd have to wait for the frConstraint's to be linked to the odb Rules.

Reply:
We can take this up in a separate PR.